### PR TITLE
Merge bottom banner

### PR DESCRIPTION
**Pull request by gcater**
https://www.loom.com/share/5e84e6aba4ff414dad0e2466e8c45233?sid=20df3b7c-5f82-4316-95b5-bbfbc8e5eb53
Screenshot 2024-04-19 at 6 29 10 PM

Created a bottom Banner component and added it to section renderer. Made a slight padding adjustment to section/index.

### CODE REVIEW

File: src/components/BottomBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      nextSection = nextChapter.sections[0];
+    }
+  }
+  if (!nextSection) return <></>;
+  const nextSectionLink = nextSection
+    ? `/book/${bookId}/${nextSection.chapterId}/${nextSection.id}`
+    : "/";
+  return (
+    <div className="text-serif flex w-full flex-col pt-4">
+      <header className="jusitfy-center flex border-t border-gray-300 bg-white px-4 py-5">
```
Comment (Evanc123) [inline src/components/BottomBanner.tsx]: Typo

File: src/components/BottomBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        <div className="w-[50%]">
```
Comment (Evanc123) [inline src/components/BottomBanner.tsx]: W-1/2

File: src/components/BottomBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+          <h1 className="Black text-sm font-bold">Next Section: </h1>
```
Comment (Evanc123) [inline src/components/BottomBanner.tsx]: What is Black here?

**Review by Evanc123 — CHANGES_REQUESTED** [review]

File: src/components/BottomBanner.tsx (hunk 1 above)
Comment (gcater) [reply src/components/BottomBanner.tsx]: fixed

File: src/components/BottomBanner.tsx (hunk 2 above)
Comment (gcater) [reply src/components/BottomBanner.tsx]: cool good to know.

File: src/components/BottomBanner.tsx (hunk 3 above)
Comment (gcater) [reply src/components/BottomBanner.tsx]: No reason . took it out.

**Comment by gcater** [comment]
jusitfy->justify, w-[50%] -> w-1/2, took out Black

**Review by Evanc123 — APPROVED** [review]

**Pull request merged**